src/main/webapp/index.html

```diff
@@ -74,10 +74,30 @@ <h2>Barcode</h2>
           <p>
             Upload a barcode
           </p>
-          <p><a class="btn btn-secondary" href="#" role="button">Upload barcode photo »</a></p>
+          <p><a class="btn btn-secondary" href="#" role="button" id="upload-barcode" onclick="fileUploadDisplay()">Upload barcode photo »</a></p>
```

Comment:
How about making those elements buttons if they are not links?

Reply:
Hmm can do that.

Reply:
Done!